dvc/stages/__init__.py

```diff
@@ -171,30 +84,109 @@ def wrapper(stage, *args, **kwargs):
     return wrapper
 
 
-class Stage(object):
-    STAGE_FILE = "Dvcfile"
-    STAGE_FILE_SUFFIX = ".dvc"
+def create_stage(repo, accompany_outs=False, **kwargs):
```

Comment:
discussed with @skshetry that it is better to create `create_dvcfile` in `dvcfile.py` instead